octoprint_bettergrblsupport/__init__.py

```diff
@@ -479,15 +489,7 @@ def hook_gcode_sending(self, comm_instance, phase, cmd, cmd_type, gcode, *args,
             # relative positioning
             self.positioning = 1
 
-        #T2 # HACK:
```

Comment:
why was this hack removed?  This was put in place to address a code parsing issue specific tog code generated using T2 Laser.  Reverting it may cause harm to clients who use T2.

Reply:
See issue #7 for further reference